docs/release_notes.rst

```diff
@@ -40,6 +40,21 @@ Data Coverage
   and place in the PUDL db. For now there's a ``epacamd_eia`` output table you can use
   to merge CEMS and EIA data yourself :pr:`1692`. Eventually we'll work these crosswalk
   values into an output table combining CEMS and EIA.
+* Integrated 2021 from the :doc:`data_sources/ferc1` data. FERC updated its reporting
```

Comment:
Only think I'd add is to mention that work is ongoing and is being tracked in #1574.